pkg/utils/utils.go

```diff
@@ -422,25 +425,32 @@ func setVfsAdminMac(iface *sriovnetworkv1.InterfaceExt) error {
 	for _, addr := range vfs {
 		vfID, err := dputils.GetVFID(addr)
 		if err != nil {
-			glog.Errorf("setVfsAdminMac(): unable to get VF id %+v %q", iface.PciAddress, err)
-			return err
+			glog.Errorf("initVfs(): unable to get VF(%d) id %+v %q", vfID, iface.PciAddress, err)
+			continue
```

Comment:
Using continue instead of return is to avoid config daemon to continously restart and I don't think it's critical enough to exit the config daemon when vf init fails.